src/stru_multi/4C_stru_multi_microstatic.hpp

```diff
@@ -244,6 +249,10 @@ namespace MultiScale
 
     double density() const { return density_; };
 
+   private:
+    std::shared_ptr<Core::IO::DiscretizationVisualizationWriterMesh> micro_vtu_writer_ptr_;
```

Comment:
I think here is is not necessary to append the type `_ptr` to the name of the variable.